src/Sulu/Component/Content/Document/Subscriber/ResourceSegmentSubscriber.php

```diff
@@ -335,22 +328,22 @@ private function updateRoute($document, $generateRoutes)
      *
      * @param NodeInterface $node
      * @param string $resourceSegmentPropertyName
-     * @param string $parentPart
+     * @param string $parentUuid
      * @param string $webspaceKey
      * @param string $locale
      */
     private function updateResourceSegmentProperty(
         NodeInterface $node,
         $resourceSegmentPropertyName,
-        $parentPart,
+        $parentUuid,
         $webspaceKey,
         $locale
     ) {
         $childPart = $this->rlpStrategy->getChildPart($node->getPropertyValue($resourceSegmentPropertyName));
 
         $node->setProperty(
             $resourceSegmentPropertyName,
-            $this->rlpStrategy->generate($childPart, $parentPart, $webspaceKey, $locale)
+            $this->rlpStrategy->generate($childPart, $parentUuid, $webspaceKey, $locale)
```

Comment:
This call doesn't match the method's signature. That's probably why it is so confusing fore me. Also see the comment https://github.com/sulu/sulu/pull/2706/files#r73859768.